be/src/runtime/lake_snapshot_loader.cpp

```diff
@@ -82,6 +82,7 @@ Status LakeSnapshotLoader::_get_existing_files_from_remote(BrokerServiceConnecti
         LOG(INFO) << "finished to split files. valid file num: " << files->size();
 
     } catch (apache::thrift::TException& e) {
+        (void)client.reopen(config::thrift_rpc_timeout_ms);
```

Comment:
should this reopen code change be split into a separate PR? It has nothing to do with the rpc helper refactor.

Reply:
It is necessary for them, this PR will be backported to other branches in the future.

Reply:
I mean it's fine to add the reopen logic to the codebase, but is it necessary to be in this PR? "refactor" is supposed to be no behavior change ideally

Reply:
It is easy to overlook a refactor PR, especially when the refactor PR contains changes that is not related to "REFACTOR".